scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/entity/ScheduleEntity.java

```diff
@@ -64,6 +74,14 @@
 	@Column(name = "initial_min_instance_count")
 	@JsonProperty(value = "initial_min_instance_count")
 	private Integer initialMinInstanceCount;
+	
+	@Column(name = "updated_at")
+	@JsonProperty(value = "updated_at")
+	@JsonFormat(pattern = DateHelper.DATE_TIME_FORMAT)
+	@JsonDeserialize(using = DateTimeDeserializer.class)
+	@JsonSerialize(using = DateTimeSerializer.class)
+	private LocalDateTime updatedAt;

```

Comment:
It is just for internal use and will not be exposed.  So in my opinion "@ApiModelProperty" is not need.